python2/koans/about_control_statements.py

```diff
@@ -73,7 +73,7 @@ def test_for_statement_with_tuples(self):
             result.append("Contestant: '" + knight + \
             "'   Answer: '" + answer + "'")
 
-        text = __
+        text = "Contestant: 'Robin'   Answer: 'Blue! I mean Green!'"
```

Comment:
By the way, maybe you already know, but "Python" was named after the Monty Python comedy troupe, and there are plenty of references to the Monty Python films throughout the language, books, and exercises.

The knights and answers from the list are from this iconic scene from "Monty Python and the Holy Grail": https://www.youtube.com/watch?v=dPOyOM7wxlE